.github-comment.yaml

```diff
@@ -0,0 +1,19 @@
+exec:
+  gofmt:
+    - when: ExitCode != 0
+      template: |
+        ## :x: `gofmt -s` failed
+        The following files are not formatted. Please run `gofmt -s -w .` and commit.
+
+        ```
+        {{.CombinedOutput}}
+        ```
```

Comment:
`ExitCode != 0` indicates that gofmt didn't exit successfully. In that case the following comment will be posted to the PR.
![CleanShot 2024-06-19 at 11 50 35](https://github.com/signalsciences/terraform-provider-sigsci/assets/29038315/e88c4ca4-4f9a-4634-b7e8-874135888b21)